ballista/scheduler/src/state/task_manager.rs

```diff
@@ -524,24 +524,35 @@ impl<T: 'static + AsLogicalPlan, U: 'static + AsExecutionPlan> TaskManager<T, U>
     pub(crate) async fn launch_multi_task(
         &self,
         executor: &ExecutorMetadata,
-        tasks: Vec<Vec<TaskDescription>>,
+        tasks: HashMap<(String, usize), Vec<TaskDescription>>,
```

Comment:
do we need to change this parameter? if it is just job_id that we need, we can get that from `task.partition` if im not mistaken